back/backend/lib/auth.py

```diff
@@ -0,0 +1,17 @@
+from flask import session
+
+from backend.models.user import User
+
+
+def logged_in():
+    return 'wca_account_number' in session
+
+
+def user():
+    if not logged_in():
+        return None
+
+    wca_account_number = session['wca_account_number']
+
+    user = User.get_by_id(wca_account_number)
```

Comment:
I don't really have any expierence with Google data store, but should we consider what happens here if the account isn't found, or is this guaranteed not to happen by this point?